### PR TITLE
Flattie/pass chat!

### PR DESCRIPTION
**Pull request by iatkins**

### CODE REVIEW

File: server/proxy/server.js [hunk 1]
```diff
@@ -59,6 +58,15 @@ app.get('/connect', (req, res) => {
     }); 
 });
 
+app.get('/send-message', (req,res) => {
```
Comment (tillywoodfield) [inline server/proxy/server.js]: `(req, res)`

File: src/components/Chat/Chat.module.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
+.chatRoom {
+    font-size: 1em;
```
Comment (tillywoodfield) [inline src/components/Chat/Chat.module.css]: I think `font-size: 1em` doesn't do anything, it just says make it the current size

File: src/components/Chat/Chat.module.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    margin: 0em 0.5em 0.5em 0.5em;
+    letter-spacing: 0.2em;
+}
+
+.chatBox {
+    font-size: 1em;
+    display: flex;
+    flex-grow: 1;
+    flex-direction: column;
+    max-height: 35vh;
```
Comment (tillywoodfield) [inline src/components/Chat/Chat.module.css]: In the other pr I thought you got it to scroll properly without fixing the value?

File: cypress/integration/chat_spec.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+describe('on send a message', () => {
+    beforeEach(() => {
+        cy.login('Bob');
+        cy.sendMessage();   
+    });
+
+    it('displays the message', () => {     
+        cy.get('[data-cy="messages"]').should('have.text', 'Bob : message');
+    })
```
Comment (sibela46) [inline cypress/integration/chat_spec.js]: semicolon

File: cypress/integration/chat_spec.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    it('clears the input', () => {
+        cy.get('[data-cy="type-message"]').should('be.empty');
+    });
+});
+
+describe('on someone else sends a message', () => {
+    beforeEach(() => {
+        cy.request(`http://localhost:12346/connect?url=${encodeURIComponent(url)}`)
+        .then(() => cy.login('Bob'));
+        cy.createRoom(3);
```
Comment (sibela46) [inline cypress/integration/chat_spec.js]: you should chain these with 'then'

File: cypress/integration/instructions_spec.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ const url = Cypress.config().baseUrl;
 describe('Waiting', () => {
     beforeEach(() => cy.login('unicorn'));
 
-    describe('when first player is playing card and word', () => {
+    describe('when storyteller is playing card and word', () => {
         beforeEach(() => {
             cy.createRoom(3);
             cy.request(`http://localhost:12346/connect?url=${encodeURIComponent(url)}`)
```
Comment (sibela46) [inline cypress/integration/instructions_spec.js]: one more then here?

File: cypress/integration/instructions_spec.js [hunk 7]
```diff
@@ -13,7 +13,7 @@ describe('Waiting', () => {
             });
```
Comment (sibela46) [inline cypress/integration/instructions_spec.js]: this could be a single line:
.then(() => cy.startGame());

File: server/services/socket.js [hunk 8]
```diff
@@ -13,18 +13,25 @@ exports.setupSocket = (server, session) => {
...
+                const { username, message } = msg;                
+                emitMessage(username, message);
+            });
         } else {
             socket.disconnect();
         }
         socket.on('disconnect', disconnected => {
             sockets = sockets.filter(socket => socket !== disconnected);
-        });
+        });
```
Comment (sibela46) [inline server/services/socket.js]: whitespace

**Review by sibela46 — DISMISSED** [review]

**Review by sibela46 — APPROVED** [review]

**Pull request merged**